### PR TITLE
Lamp handler

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: src/LampHandler.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+function LampHandler() {
+  const [lamp, setLamp] = useState(false);
+
+  const handleClick = () => {
+    setLamp((current) => !current);
```
Comment (blacksrc) [inline src/LampHandler.jsx]: Nice 🔥 Can't you just write?
```suggestion
    setLamp(!lamp);
```

File: src/LampHandler.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+function LampHandler() {
+  const [lamp, setLamp] = useState(false);
```
Comment (blacksrc) [inline src/LampHandler.jsx]: Better naming:
```suggestion
  const [isOn, setIsOn] = useState(false);
```

File: src/LampHandler.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  const handleClick = () => {
+    setLamp((current) => !current);
+  };
+
+  return (
+    <>
+      <button onClick={handleClick}>TURN OFF/ON</button>
```
Comment (blacksrc) [inline src/LampHandler.jsx]: Please change the text accordingly.

Comment (hedita) [reply src/LampHandler.jsx]: Done.

**Pull request merged**